src/components/tools/Stats/MonthTiming.svelte

```diff
@@ -24,17 +26,24 @@
   export let periodList;
 
   /** Id of default group to initialize StatPanel. This prop is optional. Defaults to first item in groupList */
-  export let groupId;
+  export let groupId = groupList[0].id;
```

Comment:
Same here, would it make sense to provide a default value for `periodList`?

Reply:
@clhenrick The constants `DEFAULT_STAT_GROUPS` and `DEFAULT_STAT_PERIODS` from `tools/_common/constants` can be imported as set a default values.

Reply:
fixed via https://github.com/berkeley-gif/caladapt-website-2021/pull/128/commits/4add49f56dc12f2c1cefb201e7f6168a8081ac54